function-aws/src/test/groovy/io/micronaut/function/aws/AfterExecutionEventListener.java

```diff
@@ -0,0 +1,25 @@
+package io.micronaut.function.aws;
+
+import io.micronaut.context.event.ApplicationEventListener;
+import io.micronaut.function.aws.event.AfterExecutionEvent;
+import jakarta.inject.Singleton;
+
+@Singleton
+public class AfterExecutionEventListener implements ApplicationEventListener<AfterExecutionEvent> {
```

Comment:
Please, delete it. Each test has its own `static class` annotated with `@Requires(property = 'spec.name'` to avoid bean pollution.

Reply:
resolved via https://github.com/micronaut-projects/micronaut-aws/pull/1433/commits/267a5a6c89a08bdaacbe7161f11efd498c3fb3ee